apps/civicsignalblog/migrations/20241122_054517_migration.ts

```diff
@@ -0,0 +1,50 @@
+import { MigrateUpArgs, MigrateDownArgs } from "@payloadcms/db-mongodb";
+import mongoose from "mongoose";
+
+const mongoURL = process.env.MONGO_URL;
+const oldGlobalConfigType = "settings-site";
+const newGlobalConfigType = "settings-research-site";
+
+export async function up({ payload }: MigrateUpArgs): Promise<void> {
+  await mongoose.connect(mongoURL);
+  const db = mongoose.connection.db;
+
+  const existingGlobalConfig = await db.collection("globals").findOne({
+    globalType: oldGlobalConfigType,
+  });
+
+  if (!existingGlobalConfig) {
+    payload.logger.error(
+      `No existing global settings [${oldGlobalConfigType}] config found, exiting...`,
+    );
+    return;
+  }
+
+  let newGlobalConfig = await db.collection("globals").findOne({
+    globalType: newGlobalConfigType,
+  });
+
+  if (newGlobalConfig) {
+    payload.logger.error(
+      `Migration failed: Unable to migrate [${oldGlobalConfigType}] to [${newGlobalConfigType}] because a global configuration with this name already exists.`,
+    );
+    return;
+  }
```

Comment:
```suggestion
  const existingNewGlobalConfig = await db.collection("globals").findOne({
    globalType: newGlobalConfigType,
  });

  if (existingNewGlobalConfig) {
    payload.logger.error(
      `Migration 20241122_054517 failed: Global type [${newGlobalConfigType}] already exists.`,
    );
    return;
  }
```

Just used `20241122_054517` from filename as migration name but if there is a better name, lets use that.

Reply:
Filename looks good, we can easily trace which migration failed👍